dipy/viz/actor.py

```diff
@@ -175,7 +175,7 @@ def copy(self):
             im_actor = ImageActor()
             im_actor.input_connection(self.output)
             im_actor.SetDisplayExtent(*self.GetDisplayExtent())
-            im_actor.opacity(opacity)
+            im_actor.opacity(self.GetOpacity())
             im_actor.tolerance(picking_tol)
```

Comment:
While you're at it, can you also fix this same issue for the tolerance?

Reply:
done